Makefile.one

```diff
@@ -175,59 +230,46 @@ help:
 	@echo "BTFHUB=1 STATIC=1 make ...  # build static binaries, embed BTF"
 	@echo ""
 
+#
 # variables
+#
 
 BPF_VCPU = v2
 
-# requirements
-
-.PHONY: reqs_cmds
-reqs_cmds: \
-	check_$(CMD_GIT) \
-	check_$(CMD_CLANG) \
-	check_$(CMD_LLC) \
-	check_$(CMD_STRIP) \
-	check_$(CMD_RM) \
-	check_$(CMD_GREP) \
-	check_$(CMD_TOUCH) \
-	check_$(CMD_CAT) \
-	check_$(CMD_MD5) \
-	check_$(CMD_INSTALL) \
-	check_$(CMD_MKDIR) \
-	check_$(CMD_PKGCONFIG) \
-	check_$(CMD_GO) \
-	check_$(CMD_OPA)
-
-.PHONY: reqs_libs
-reqs_libs: \
-	checklib_$(LIB_ELF) \
-	checklib_$(LIB_ZLIB)
-
-.PHONY: reqs
-reqs:
-	$(MAKE) reqs_cmds
-	$(MAKE) reqs_libs
-
+#
 # output dir
+#
 
 OUTPUT_DIR = ./dist
 
-$(OUTPUT_DIR):
-	$(CMD_MKDIR) -p $@
-	$(CMD_MKDIR) -p $@/libbpf
-	$(CMD_MKDIR) -p $@/libbpf/obj
+$(OUTPUT_DIR): \
+	.check_$(CMD_MKDIR)
+	@#
+	@$(CMD_MKDIR) -p $@
+	@$(CMD_MKDIR) -p $@/libbpf
+	@$(CMD_MKDIR) -p $@/libbpf/obj
 
+#
 # embedded btfhub
+#
 
-$(OUTPUT_DIR)/btfhub:
-	$(CMD_MKDIR) -p $@
-	$(CMD_TOUCH) $@/.place-holder # needed for embed.FS
+$(OUTPUT_DIR)/btfhub: \
+	.check_$(CMD_MKDIR) \
+	.check_$(CMD_TOUCH)
+	@#
```

Comment:
I've chosen this formatting style so we can always split the logic: build target dependency VS build commands. The option of having each dependency on its own line is to remove burden from maintenance (patches, for example, would change single lines for adding or removing build dependencies, etc). This is a well known strategy for sustaining engineering.